zerver/openapi/zulip.yaml

```diff
@@ -6397,10 +6415,38 @@ paths:
                         <td><code>wildcard_mentioned</code></td>
                         <td>
                             Whether this message contained
-                            <a href="/help/mention-a-user-or-group#mention-everyone-on-a-stream">wildcard mention</a>
+                            <a href="/help/mention-a-user-or-group#mention-everyone-on-a-stream">stream wildcard mention</a>
                             like @**all**. Cannot be changed by the user directly, but
                             can change if the message is edited to add/remove
-                            a wildcard mention.
+                            a stream wildcard mention.
```

Comment:
I think this documentation doesn't match my understanding of the decision, which is that this compatible `wildcard_mentioned` would be true 

Also, we always want API documentation to be written from the mindset of clearly and explicitly answering the questions that one needs to answer in order to do a correct client implementation. 

> In Zulip 8.0 (feature level 215), the `wildcard_mentioned` flag was deprecated in favor of the `stream_wildcard_mentioned` and `topic_wildcard_mentioned` flags. The `wildcard_mentioned` flag exists for backwards-compatibility with older clients, and equals `stream_wildcard_mentioned || topic_wildcard_mentioned`. Clients interacting with older servers should treat this field as an old name for `stream_wildcard_mentioned`; `@topic` mentions were not available prior to this feature level.